libraries/Microsoft.Bot.Builder.Ai.QnA/Microsoft.Bot.Builder.Ai.QnA.csproj

```diff
@@ -42,10 +42,9 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.Cognitive.LUIS" Version="2.0.2" />
     <PackageReference Include="Microsoft.Bot.Builder" Condition=" '$(PackageVersion)' == '' " Version="4.0.0-local" />
-    <PackageReference Include="Microsoft.Recognizers.Text" Version="1.0.0.35" />
```

Comment:
>  <PackageReference Include="Microsoft.Recognizers.Text" Version="1.0.0.35" /> [](start = 2, length = 78)

I think there is supposed to be another reference here like this:
  <PackageReference Include="Microsoft.Bot.Builder" Condition=" '$(PackageVersion)' != '' " Version="$(PackageVersion)" />
Check other project files